src/main/java/com/hubspot/jinjava/util/WhitespaceUtils.java

```diff
@@ -142,5 +143,16 @@ public static String unwrap(String s, String prefix, String suffix) {
     return s.substring(start + prefix.length(), end - suffix.length() + 1);
   }
 
+  @Nullable
+  public static StringBuilder quoteIfNotNull(CharSequence charSequence) {
```

Comment:
where is this used?

Reply:
It's not actually used in this repository, but it's a helper method for classes that extend PyishSerializable. Since all the other logic for checking if strings are quoted and removing quotes from strings lives here, I figured it would be nice to have the reverse